content/en/agent/troubleshooting/agent_check_status.md

```diff
@@ -59,6 +59,14 @@ If your issue continues, [reach out to the Datadog support team][1] with a [flar
 
 Run the following script, with the proper `<CHECK_NAME>`:
 
+For Agent versions >= 6.12:
+
+```
+C:\Program Files\Datadog\Datadog agent\bin\agent.exe check <CHECK_NAME>
```

Comment:
Should Agent be capitalized?
`C:\Program Files\Datadog\Datadog agent`

If yes, update lines 65 and 71.
If no, update line 96 and line 135 of `content/en/developers/dogstatsd/datagram_shell.md`.

Reply:
both work actually, but I'll make things consistent

Reply:
addressed